### PR TITLE
Update homepage fake data

### LINKED ISSUE #4052: Update Homepage fake data
image

Let's update homepage's fake data so we can make the fake homepage slightly realistic.

1. [ ] add some fake blog posts to blog section
2. [ ] highlights added by factory should be at most 5 (currently factory adds 6 highlights which exceeds the max number of highlights the homepage can have)

### PR DESCRIPTION
**Pull request by mmmavis**
Closes #4052

### CODE REVIEW

File: network-api/networkapi/wagtailpages/factory/homepage_features.py [hunk 1]
```diff
@@ -21,35 +22,38 @@ class Meta:
...
+    featured_blogs = BlogPage.objects.all()[:NUM_BLOGS]
+    featured_highlights = [HighlightFactory.create() for i in range(NUM_HIGHLIGHTS)]
+
+    home_page.featured_blogs = [
+        HomepageFeaturedBlogsFactory.build(blog=featured_blogs[i]) for i in range(NUM_BLOGS)
     ]
-    home_page.featured_news = [
-        HomepageFeaturedNewsFactory.build(news=featured_news[i]) for i in range(6)
+    home_page.featured_highlights = [
+        HomepageFeaturedHighlightsFactory.build(highlight=featured_highlights[i]) for i in range(NUM_HIGHLIGHTS)
```
Comment (Pomax) [inline network-api/networkapi/wagtailpages/factory/homepage_features.py:56]: We're running list comprehension twice here, but we can do the same thing without the `featured_highlights` intermediary:

```
home_page.featured_highlights = [
    HomepageFeaturedHighlightsFactory.build(
        highlight=HighlightFactory.create()
    )
    for i in range(NUM_HIGHLIGHTS)
]
```

File: network-api/networkapi/wagtailpages/factory/homepage_features.py [hunk 2]
```diff
@@ -21,35 +22,38 @@ class Meta:
...
+    NUM_BLOGS = 4
+    NUM_HIGHLIGHTS = 5
 
-    home_page.featured_highlights = [
-        HomepageFeaturedHighlightsFactory.build(highlight=featured_highlights[i]) for i in range(6)
+    featured_blogs = BlogPage.objects.all()[:NUM_BLOGS]
+    featured_highlights = [HighlightFactory.create() for i in range(NUM_HIGHLIGHTS)]
+
+    home_page.featured_blogs = [
+        HomepageFeaturedBlogsFactory.build(blog=featured_blogs[i]) for i in range(NUM_BLOGS)
```
Comment (Pomax) [inline network-api/networkapi/wagtailpages/factory/homepage_features.py:53]: We're running list comprehension twice here, but we can do the same thing without the `featured_blogs` ntermediary:

```
home_page.featured_blogs = [
    HomepageFeaturedBlogsFactory.build(
        blog=BlogPage.objects.all()[i]
    )
    for i in range(NUM_BLOGS)
]
```

File: network-api/networkapi/wagtailpages/models.py [hunk 3]
```diff
@@ -10,8 +10,8 @@
     Styleguide,
     Homepage,
     PeoplePage,
+    HomepageFeaturedBlogs,
     HomepageFeaturedHighlights,
-    HomepageFeaturedNews,
```
[the remark this answers is not in the document]
Comment (Pomax) [reply network-api/networkapi/wagtailpages/models.py:14]: let's remove it, but do all that work in a single issue, so let's not remove these lines here right now.

[the remark this answers is not in the document]
Comment (mmmavis) [reply network-api/networkapi/wagtailpages/models.py:14]: k ticket filed!

File: network-api/networkapi/wagtailpages/factory/homepage_features.py (hunk 2 above)
Comment (mmmavis) [reply network-api/networkapi/wagtailpages/factory/homepage_features.py:53]: Wow! Will update

Comment (mmmavis) [reply network-api/networkapi/wagtailpages/factory/homepage_features.py:53]: Should we store `BlogPage.objects.all()` as a variable before this line?

Comment (Pomax) [reply network-api/networkapi/wagtailpages/factory/homepage_features.py:53]: Django should already cache it internally, so we should be fine for the load_fake_data purposes

**Pull request merged**